exegol/utils/DockerUtils.py

```diff
@@ -446,7 +447,8 @@ def downloadImage(cls, image: ExegolImage, install_mode: bool = False) -> bool:
         logger.info(f"{'Installing' if install_mode else 'Updating'} exegol image : {image.getName()}")
         name = image.updateCheck()
         if name is not None:
-            logger.info(f"Starting download. Please wait, this might be (very) long.")
+            logger.raw(f"[bold blue][*][/bold blue] Pulling compressed image, starting a [cyan1]{image.getDownloadSize()}[/cyan1] download :satellite:{os.linesep}", level=logging.INFO, markup=True, emoji=True)
```

Comment:
Why do you need to use `raw` method ? If it's for handling the `~` char, couldn't you just do `\~` ?

Reply:
Indeed, mistake on my end, I had to do `logger.raw` for something else during my dev, which I removed later on. And I left it like this but `logger.info` should work fine. Fixing now

Reply:
Don't forget to remove your `import logging` at the beginning of the file too, it shouldn't be necessary now